app/uk/gov/hmrc/cdsreimbursementclaimfrontend/models/DutiesSelectedAnswer.scala

```diff
@@ -16,60 +16,40 @@
 
 package uk.gov.hmrc.cdsreimbursementclaimfrontend.models
 
+import cats.data.NonEmptyList
 import cats.kernel.Eq
 import julienrf.json.derived
-import play.api.libs.json.OFormat
-import uk.gov.hmrc.cdsreimbursementclaimfrontend.models.form.{DutiesSelected, Duty}
+import play.api.libs.json.{Format, OFormat}
+import uk.gov.hmrc.cdsreimbursementclaimfrontend.models.form.Duty
 
-sealed trait DutiesSelectedAnswer extends Product with Serializable
+final case class DutiesSelectedAnswer(duties: NonEmptyList[Duty]) extends AnyVal
```

Comment:
How do you think would it be better to have a package object containing all answers as aliases. For example:

```
package uk.gov.hmrc.cdsreimbursementclaimfrontend.model

package object answers {

  type DutiesSelectedAnswers = NonEmptyList[Duty]
  type MaybeSomeAnotherAnswer = Option[Boolean]
  
  object DutiesSelectedAnswers {
    def apply(duty: Duty) = NonEmptyList.head(duty)
  }
  // This will simplify 
  // DutiesSelectedAnswer(NonEmptyList.of(Duty(TaxCode.A00)))
  // to
  // val d = DutiesSelectedAnswer(Duty(TaxCode.A00))
  // and then you can do
  // d +: Duty("Ireland one") or d ++ List(Duty("Wales one"))
  

  //--formats--
  // Can be moved elsewhere, but good to have defined once and reused over everywhere

  def nelReads[A : Reads]: Reads[NonEmptyList[A]] =
    Reads.of[List[A]].collect(JsonValidationError("Expected a non empty list but got an empty list")) { case x :: xs =>
      NonEmptyList(x, xs)
    }

  def nelWrites[A : Writes]: Writes[NonEmptyList[A]] =
    Writes.of[List[A]].contramap(_.toList)

  implicit def nelFormat[A : Format]: Format[NonEmptyList[A]] =
    Format(nelReads, nelWrites)
}

``` 
The thing it would not be required to define OFormats for each

Reply:
Not sure about `MaybeSomeAnotherAnswer` but otherwise definitely.